ci/retry/retry

```diff
@@ -17,7 +17,7 @@ __log_out() {
   echo "$1" 1>&2
 }
 
-# Paramters: max_tries min_sleep max_sleep constant_sleep fail_script EXECUTION_COMMAND
+# Parameters: max_tries min_sleep max_sleep constant_sleep fail_script EXECUTION_COMMAND
```

Comment:
This is an upstream file, please don't modify it.

Reply:
Are we really expecting to sync this file with the upstream? Seems like a pretty trivial script and it hasn't been updated for two years in the upstream repo. I suggest we treat it as an ordinary in-repo shell script :)

Reply:
Upstream was updated [<2 months ago](https://github.com/kadwanev/retry/commit/7d2caa94a7e54bd15f54fb7f2395e11b2b829575). If we're going to fix the typos in this file, then we should at least fix all of them so that we only have to do this once.

Reply:
Oh, I misread "2 months" as "2 years". Sorry: upstream seems to be alive :)

Reply:
@RandyMcMillan Would you mind reporting the `retry` typo upstream to https://github.com/kadwanev/retry? :)

Reply:
Thanks for the feedback.